### PR TITLE
Removed unit test for prerelease version and VSCode temp folder

### PR DESCRIPTION
**Pull request by Gijsreyn**
- [x] I have signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-dsc).
- [ ] This pull request is related to an issue.

-----

Addresses faulty test in Dotnet DSC resource and support for cross-platform VSCode.
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-dsc/pull/129)

### CODE REVIEW

**Comment by Gijsreyn** [comment]
@ryfu-msft There you go. Small one. I just removed the testcase because we are already validating it with PowerShell.

File: resources/Microsoft.VSCode.Dsc/Microsoft.VSCode.Dsc.psm1 [hunk 1]
```diff
@@ -95,8 +95,8 @@ function Invoke-VSCode {
         [string]$Command
     )
 
-    $stdErrTempFile = "$env:TEMP\$((New-Guid).Guid)"
-    $stdOutTempFile = "$env:TEMP\$((New-Guid).Guid)"
+    $stdErrTempFile = [System.IO.Path]::GetTempPath() + (New-Guid).Guid
+    $stdOutTempFile = [System.IO.Path]::GetTempPath() + (New-Guid).Guid
```
Comment (ryfu-msft) [inline resources/Microsoft.VSCode.Dsc/Microsoft.VSCode.Dsc.psm1:99]: Now that I think about it, string concatenation for path is probably not the way to go. Can you change this to use `Join-Path`?

**Review by ryfu-msft — CHANGES_REQUESTED** [review]

File: resources/Microsoft.VSCode.Dsc/Microsoft.VSCode.Dsc.psm1 (hunk 1 above)
Comment (Gijsreyn) [reply resources/Microsoft.VSCode.Dsc/Microsoft.VSCode.Dsc.psm1]: Nitter. Just updated it.

**Comment by Gijsreyn** [comment]
Please run it again.

**Review by ryfu-msft — APPROVED** [review]

**Pull request merged**